docs/users/json.rst

```diff
@@ -206,7 +206,7 @@ The following optional elements may also be included in the ``additional_info``
      - array of strings
      - A list of MusicBrainz Work IDs that may be associated with this recording.
    * - ``tracknumber``
-     - integer
+     - string
```

Comment:
Looks good, listen submissions with alphanumeric track numbers are accepted and displayed fine.
I already updated my submission client to make use of this 😎